### PR TITLE
[ENH] Add Twitter badge to README and link to website to landing page

### LINKED ISSUE #266: Add twitter badge to gh and website
Adding a BIDS twitter badge/reference to the:

- [x] github readme
- [x] RTD rendering (probably on the index)
- [x] BIDS .io website

Please feel free to add other locations our twitter badge can be

Edit: https://github.com/bids-standard/bids-specification/pull/268 addresses point 1 and 2
https://github.com/bids-standard/bids-website/pull/49 addresses point 3

### PR DESCRIPTION
**Pull request by franklin-feingold**
addresses https://github.com/bids-standard/bids-specification/issues/266 point 1 and 2 

this will help raise awareness of our twitter. also added the mailing list sign up link to the RTD index

closes #266

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.com/bids-standard/bids-specification.svg?branch=master)](https://travis-ci.com/bids-standard/bids-specification)
 [![CircleCI](https://circleci.com/gh/bids-standard/bids-specification.svg?style=svg)](https://circleci.com/gh/bids-standard/bids-specification)
+![](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)
```
Comment (effigies) [inline README.md]:
```suggestion
[![@BIDSstandard](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)](https://twitter.com/BIDSstandard]
```

File: src/index.md [hunk 2]
```diff
@@ -7,3 +7,8 @@ about the core specification, as well as many modality-specific extensions.
 To get started, [check out the introduction](01-introduction.md). If you'd like
 more information on how to adapt your own datasets to match the BIDS
 specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+
+If you are interested in getting occasional updates on BIDS 
+please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit)
+
+You can also find us on Twitter! ![](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)
```
Comment (effigies) [inline src/index.md]: A badge here doesn't seem to add anything.

```suggestion
You can also find us on [Twitter](https://twitter.com/BIDSstandard]!
```

File: src/index.md [hunk 3]
```diff
@@ -7,3 +7,8 @@ about the core specification, as well as many modality-specific extensions.
 To get started, [check out the introduction](01-introduction.md). If you'd like
 more information on how to adapt your own datasets to match the BIDS
 specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+
+If you are interested in getting occasional updates on BIDS 
+please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit)
```
Comment (effigies) [inline src/index.md]:
```suggestion
please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit).
```

Does the mailing list not have a more standard subscription management? e.g., most listservs have a page where you can sign up and then go back and unsubscribe if you want. Using a Google Form makes it feel like it's going to just be a list in somebody's contacts, which is hard to get off of.

File: README.md (hunk 1 above)
Comment (franklin-feingold) [reply README.md]: I applied this change on my test directory and the rendering may not be what was intended - https://github.com/franklin-feingold/bids-specification-test/blob/master/README.md

Here is the previous rendering on my fork - https://github.com/franklin-feingold/bids-specification/blob/readme/README.md

Comment (effigies) [reply README.md]: I see. I don't know how to make this shield work, then, but it doesn't seem very useful to create one that doesn't link to the Twitter account.

File: src/index.md (hunk 3 above)
Comment (franklin-feingold) [reply src/index.md]: at this time no. having a more standard subscription management can be good at this point before using it (e.g. newsletters)

May you have recommendations on one?

File: src/index.md (hunk 2 above)
Comment (franklin-feingold) [reply src/index.md]: I think a badge looks better than having a hyperlink to twitter

File: README.md (hunk 1 above)
Comment (effigies) [reply README.md]: Oh, there was a typo in my suggestion. Fixed.

**Comment by franklin-feingold** [comment]
Adding the badge and mailing list on the index will be strongly beneficial in providing what our information distribution channels are and having them be fully utilized. Take the case of a new member of the community that recently learned about BIDS: they may want to learn more through reading the specification and potentially wanting to stay in the loop (signing up for the mailing list/following on twitter). We want to make it as easy as possible to find where information is distributed. This aims at lowering the barriers and making it more apparent. This was another takeaway I had from the BIDS community meeting at OHBM. Some information is not promoted/distributed well and this has a downstream effect on contributing and staying in the loop.

If this is not placed in the index - it should be placed somewhere in the specification to provide this opportunity. It can be difficult to get further into the loop and placing our information distribution channels in better locations can help the community contribute/stay up to date on the project. 

I understand the position of keeping it clean but this decision will not then make it become "a place of many links" location like the website. I think these are both important to promote, have a strong upside, and are valuable to the continued growth and outreach.

Why do you think this would adversely hurt the specification?

File: README.md (hunk 1 above)
Comment (franklin-feingold) [reply README.md]: looks like it works now, thank you!

File: src/index.md (hunk 2 above)
Comment (franklin-feingold) [reply src/index.md]: applying your previous suggestion to get the badge directly linked to twitter

```suggestion
You can also find us on Twitter! [![@BIDSstandard](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)](https://twitter.com/BIDSstandard)
```

**Comment by sappelhoff** [comment]
> Adding the badge and mailing list on the index will be strongly beneficial in providing what our information distribution channels are and having them be fully utilized.

I think I disagree with the "strongly beneficial" --> it is true that there is some benefit, but it comes at the cost of cluttering the specification.

> Take the case of a new member of the community that recently learned about BIDS: they may want to learn more through reading the specification and potentially wanting to stay in the loop (signing up for the mailing list/following on twitter). 

Yes, fair point. I think that we can obtain that benefit through linking to our [bids.neuroimaging.io](bids.neuroimaging.io) page. See this mockup: `For more information on the BIDS ecosystem in general, visit the [BIDS homepage](bids.neuroimaging.io).`

> We want to make it as easy as possible to find where information is distributed. This aims at lowering the barriers and making it more apparent.

Yes, but having links everywhere does not mean that it's getting easier to navigate the BIDS ecosystem. Instead, we should have a coherent overview page like

- our homepage
- the overview page that is [work in progress](https://docs.google.com/document/d/1u680oW6ktEXKyE6_V3kcdZAZ5Ma445ZwG9_oBuE7z1A/edit)

and we should consistently link to the overviews. And then from the overviews back to the specialized resources. This avoids duplication, and thereby reduces the chances of errors and confusion. (this answers your question `Why do you think this would adversely hurt the specification?`)

> This was another takeaway I had from the BIDS community meeting at OHBM. Some information is not promoted/distributed well and this has a downstream effect on contributing and staying in the loop.

I absolutely agree with you, I would just employ a different method, as argued above. I think we will win if each document is clean and clear with its purpose ... with a single link to an overview, from where we get to other documents that are again clean and clear.

**Comment by franklin-feingold** [comment]
> Yes, fair point. I think that we can obtain that benefit through linking to our bids.neuroimaging.io page. See this mockup: For more information on the BIDS ecosystem in general, visit the [BIDS homepage](bids.neuroimaging.io).

We can test putting the neuroimaging.io webpage in this place and evaluate. This can be iterated on in the future and put the [centralized resources](https://docs.google.com/document/d/1u680oW6ktEXKyE6_V3kcdZAZ5Ma445ZwG9_oBuE7z1A/edit) final figure in its place. For now, can see how it works in practice and if it is accomplishing this goal of reducing the barriers and making it easier to navigate the terrain. 

> and we should consistently link to the overviews. And then from the overviews back to the specialized resources. This avoids duplication, and thereby reduces the chances of errors and confusion.

Yes, that puts the responsibility on those resources. I agree with the process. Probably having just one main central resource to help continue avoiding confusion. 

As an aside: I think duplication/redundancy if done correctly will help the system function more robustly. It is being strategic and putting important information in high potential areas to help with learning/navigating. The errors and confusion I think arises if you are sending to different places or if it does not fit. If it is being funneled to the same overview that can help with the diffusion you are describing. It creates checks and balances within the system. I think something to consider and think about, no action here.

> I think we will win if each document is clean and clear with its purpose ... with a single link to an overview, from where we get to other documents that are again clean and clear.

That sounds good. I see your point with keeping this focused and directed approach on that particular document. My concern is that then it will be taken in a vacuum and seeing how it works with the rest of the system could be difficult. This can be the responsibility of the individual documents to help place it into context of the rest of the system. Food for thought

File: src/index.md (hunk 3 above)
Comment (franklin-feingold) [reply src/index.md]:
```suggestion

```

File: src/index.md (hunk 2 above)
Comment (franklin-feingold) [reply src/index.md]:
```suggestion

```

**Comment by sappelhoff** [comment]
@franklin-feingold could it be that you have the option "allow edits from maintainers" disabled?

see: https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork

I cannot commit your most recent suggestions.

**Comment by franklin-feingold** [comment]
hmm... nope I have the `allow edits from maintainers` on
Screen Shot 2019-07-16 at 9 56 32 AM

I committed the recent suggestions

File: src/index.md [hunk 4]
```diff
@@ -7,3 +7,7 @@ about the core specification, as well as many modality-specific extensions.
 To get started, [check out the introduction](01-introduction.md). If you'd like
 more information on how to adapt your own datasets to match the BIDS
 specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+
+If you are interested in getting occasional updates on BIDS
```
Comment (effigies) [inline src/index.md]: Sentence fragment. Guessing it was supposed to be removed.
```suggestion
```

File: src/index.md [hunk 5]
```diff
@@ -7,3 +7,6 @@ about the core specification, as well as many modality-specific extensions.
 To get started, [check out the introduction](01-introduction.md). If you'd like
 more information on how to adapt your own datasets to match the BIDS
 specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+
+For an overview of the BIDS ecosystem, visit the [BIDS homepage](bids.neuroimaging.io).
```
Comment (franklin-feingold) [inline src/index.md]:
```suggestion
For an overview of the BIDS ecosystem, visit the [BIDS homepage](https://bids.neuroimaging.io).
```

File: src/index.md [hunk 6]
```diff
@@ -7,3 +7,6 @@ about the core specification, as well as many modality-specific extensions.
 To get started, [check out the introduction](01-introduction.md). If you'd like
 more information on how to adapt your own datasets to match the BIDS
 specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+
+For an overview of the BIDS ecosystem, visit the [BIDS homepage](https://bids.neuroimaging.io).
+
```
Comment (franklin-feingold) [inline src/index.md]:
```suggestion
```

**Pull request merged**